L/ldid/build_tarballs.jl

```diff
@@ -30,7 +30,7 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency("libplist_jll"),
-    Dependency("OpenSSL_jll"; compat="1.1.13"),
+    Dependency("OpenSSL_jll"; compat="3.0.8"),
```

Comment:
Ok, we need a newer version for aarch64-freebsd:
```suggestion
    Dependency("OpenSSL_jll"; compat="3.0.15"),
```